app.py

```diff
@@ -16,15 +14,16 @@ def book_input():
         if not text:
             flash('Text is required!')
         else:
-            result_params['title'] = title
-            result_params['text'] = text
+            session['title'] = title
```

Comment:
nit: rename to flask_session or server_session or anything that makes its purpose a bit clearer?

Reply:
It's not a variable name, it is the object itself: https://github.com/JamesLee-Jones/nlp-timeline-gen/blob/18d64111d14eca3a31189005dff862cb15dd926e/app.py#L1

I could put the line `flask_session = session`, but I feel it's a bit extra.

Reply:
Ah okay my bad! Looks good :)